### ISSUE TITLE
the directory structure should be reorganized

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#135]
The organizational structure for the source code is currently a bit odd and hard to follow.  This is because the application was highly leveraged from a Node.js example that was found on line, so the organizational structure is part PhET-like and partly derived from the example.  Things should be moved around so that the code is easier to work with.  Specifically, 
- All the JavaScript and HTML code should be under one parent directory.  As of this writing, there is a "public" directory that contains some of the code and other code is at the top level.
- The css and img directories should be at the same level as the js and html directories
- The main app file, rosetta.js, should be with the reset of the JavaScript files, not at the top level
- The routes.js file should be renamed, because it's not really setting up routes, it's defining a bunch of handlers

These are the main items that seem obvious to me, and I think I'll end up doing some other cleanup as I work through this.

**Comment by jbphet** [comment]
@mattpen - if I move rosetta.js, the systemd information will have to change to point to the new location.  I don't know how to do this.  Do you?

**Comment by mattpen** [comment]
I should be able to figure it out, what is the new location going to be?

**Comment by mattpen** [comment]
@jbphet - I think the file that needs to be changed is `/usr/lib/systemd/system/rosetta.service`.  I would just need to know what the new location of rosetta.js will be and if the working directory will change.

**Comment by jbphet** [comment]
I've done most of the reorganization, but I think I'll leave the ```rosetta.js``` file in the main directory until we cut over to the new server, since otherwise I'll need to change the ```init.d``` setup on the old server as well as the ```systemd``` configuration on the new server.  I'll leave this open and assigned to me for now.

**Comment by jbphet** [comment]
I just moved `rosetta.js` to the `js` directory and adjusted the paths.  I tested it on ox-dev and had to do a few iterations, then made the change on phet-server and tested there.  I blew the commit messages though, and attributed the commits to the wrong issue, so here they are:

+ 39d41d1cdbbb2b82405d8186a7fea6cb8b779a9c
+ c648b1879dea60aa57fa8bb12e2bf7b9076b4b2b
+ 5229681e9207f3bc4388dc2b7ed12bc1f77b2b41
+ fe01e28e05a815322b07ccea6b2b0b81a8c1b345
+ 80c19b911be00c84944c18652cb5a46dfbc08660

I'll leave this issue open for another week and, if I see translations being successfully submitted and I don't hear of any issues, I'll close it.
